style.css

```diff
@@ -43,3 +43,7 @@ body {
         display: none !important;
     }
 }
+
```

Comment:
instead of re-writing bootstrap's css, use Bootstrap differently, we found `gx-0` accomplishes the same, but is a better solution